### PR TITLE
UI reskin

### PR DESCRIPTION
**Pull request by adamgibs**
This seems to be working to me. But it may make sense for somebody else to pull it down and test it before merging into alpha though just to be sure.

### CODE REVIEW

File: src/main/cliapp/gulpfile.js [hunk 1]
```diff
@@ -1,17 +0,0 @@
...
-    .pipe(replace('.js"></script>', '.js" inline></script>'))
-    .pipe(replace('rel="stylesheet">', 'rel="stylesheet" inline>'))
-    .pipe(
-      inlinesource({
-        compress: false,
-        ignore: ["png"],
-      })
-    )
-    .pipe(gulp.dest("./build"));
-});
```
Comment (oblodgett) [inline src/main/cliapp/gulpfile.js:17]: @adamgibs I do believe we are going to need this file for building the app... would need to verify.

File: src/main/cliapp/public/assets/demo/data/customers-large.json [hunk 2]
```diff
@@ -1,3206 +1,3604 @@
-
```
Comment (oblodgett) [inline src/main/cliapp/public/assets/demo/data/customers-large.json:1]: This file probably needs to be deleted.

File: src/main/cliapp/public/assets/demo/data/customers-medium.json [hunk 3]
```diff
@@ -1,56 +1,804 @@
-
```
Comment (oblodgett) [inline src/main/cliapp/public/assets/demo/data/customers-medium.json:1]: This file probably needs to be deleted.

File: src/main/cliapp/public/assets/demo/data/events.json [hunk 4]
```diff
@@ -3,66 +3,66 @@
     	{
             "id": 1,
     		"title": "All Day Event",
-    		"start": "2023-01-03"
```
Comment (oblodgett) [inline src/main/cliapp/public/assets/demo/data/events.json:6]: This file probably needs to be deleted?

File: src/main/cliapp/public/assets/demo/data/photos.json [hunk 5]
```diff
@@ -0,0 +1,94 @@
+{
```
Comment (oblodgett) [inline src/main/cliapp/public/assets/demo/data/photos.json]: This file probably needs to be deleted?

File: src/main/cliapp/public/index.html [hunk 6]
```diff
@@ -2,13 +2,14 @@
 <html lang="en">
     <head>
         <meta charset="utf-8" />
-        <title>Alliance Curation Site</title>
+        <title>PrimeReact - Sakai</title>
```
Comment (oblodgett) [inline src/main/cliapp/public/index.html]: Probably should fix this.

**Review by oblodgett — CHANGES_REQUESTED** [review]
I think we need to go through the assets/demo directory and get rid of stuff that we don't need.... there are a lot of large pictures in there etc.

**Comment by adamgibs** [comment]
@oblodgett Yeah, that makes sense to me. Do we need anything from that directory? Seems like public/asses/demo could just be totally wiped.

**Review by oblodgett — APPROVED** [review]

**Pull request merged**